usedHttp.dart

```diff
@@ -0,0 +1,14 @@
+import 'package:http/http.dart' as http;
```

Comment:
by the way, this whole file should be bin/main.dart. Can you move this file to bin/main.dart (and replace the old one)?

Reply:
This file is moved to bin/main.dart from now.

Reply:
Can you push a a commit that fixes all the issues to this branch? The pull request will update automatically.

Reply:
Oh I see, you published the fixes to another PR. In the future, please keep the PRs separate. Each PR should do complete task. One PR should not mix several tasks or features, because it makes it harder to review. And then bugs can occur more easily.

I'll close this PR and comment on the next one.

Reply:
Oh I will make sure one PR should not mix several tasks.